pkg/namespace/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package namepsace provides management of the network namespace for
+// cilium-health agent
```

Comment:
AFAICT there's nothing in this package that is specific to `cilium-health`. It could probably be adjusted to have a generic name and implementation. I've even used the existing `spawn_netns.sh` script for non-Cilium testing locally.

Minor nit, I'd consider calling this package something like `netns` given that it's for network namespaces. It's plausible in future that we'd want to make use of other types of namespaces.

Reply:
Good idea, I will change the name to `netns` and try to make that package more universal.